app/src/test/java/java_server/FileResponderTest.java

```diff
@@ -24,6 +28,46 @@ public void returnsThePathOfTheRequestedURIFile() throws Exception {
         Assert.assertEquals("200 OK", fileResponse.getStatusCode(statusCode));
     }
 
+    @Test
+    public void returnsTheContentOfTextFile() throws Exception {
+        String uri = "text-file.txt";
+        FileResponder fileResponse = new FileResponder(testDirectory, uri);
+        byte[] fileBytes = fileResponse.getMessageBody();
+        Assert.assertEquals("file1 contents", new String(fileBytes));
+
+        Assert.assertEquals("200 OK", fileResponse.getStatusCode(statusCode));
+    }
+
+    @Test
+    public void returnsThePathOfImageJPGFile() throws Exception {
+        String uri = "image.jpeg";
+        FileResponder fileResponse = new FileResponder(testDirectory, uri);
+        byte[] fileBytes = fileResponse.getMessageBody();
+        Assert.assertTrue(Arrays.equals(Files.readAllBytes(Paths.get("/Users/victorelizalde/Documents/Github/cob_spec/public/image.jpeg")), fileBytes));
+
+        Assert.assertEquals("200 OK", fileResponse.getStatusCode(statusCode));
+    }
+
+    @Test
+    public void returnsThePathOfImagePNGFile() throws Exception {
+        String uri = "image.png";
+        FileResponder fileResponse = new FileResponder(testDirectory, uri);
+        byte[] fileBytes = fileResponse.getMessageBody();
+        Assert.assertTrue(Arrays.equals(Files.readAllBytes(Paths.get("/Users/victorelizalde/Documents/Github/cob_spec/public/image.png")), fileBytes));
+
+        Assert.assertEquals("200 OK", fileResponse.getStatusCode(statusCode));
+    }
+
+    @Test
+    public void returnsThePathOfImageGIFFile() throws Exception {
+        String uri = "image.gif";
+        FileResponder fileResponse = new FileResponder(testDirectory, uri);
+        byte[] fileBytes = fileResponse.getMessageBody();
+        Assert.assertTrue(Arrays.equals(Files.readAllBytes(Paths.get("/Users/victorelizalde/Documents/Github/cob_spec/public/image.gif")), fileBytes));
+
+        Assert.assertEquals("200 OK", fileResponse.getStatusCode(statusCode));
+    }
```

Comment:
repetición de "200 OK" – qué pasará cuando cambias a http/2? según "Single reason for change" esto se rompería